NuRadioReco/utilities/trace_utilities.py

```diff
@@ -111,6 +111,50 @@ def get_electric_field_energy_fluence(electric_field_trace, times, signal_window
 
     return f_signal * dt * conversion_factor_integrated_signal
 
+
+def get_electric_field_from_radiance(frequencies, noise_temperature, solid_angle):
```

Comment:
Thanks for doing this. Isnt a better name `get_electric_field_from_temperature`?